sourcetracker/_cli/gibbs.py

```diff
@@ -58,57 +57,75 @@
+              type=click.INT, show_default=True,
               help=('Number of independent Markov chains to grow. '
                     '`draws_per_restart` * `restarts` gives the number of '
                     'samplings of the mixing proportions that will be '
                     'generated.'))
 @click.option('--draws_per_restart', required=False, default=1,
-              type=click.INT,
+              type=click.INT, show_default=True,
               help=('Number of times to sample the state of the Markov chain '
                     'for each independent chain grown.'))
 @click.option('--burnin', required=False, default=100,
-              type=click.INT,
+              type=click.INT, show_default=True,
               help=('Number of passes (withdarawal and reassignment of every '
                     'sequence in the sink) that will be made before a sample '
                     '(draw) will be taken. Higher values allow more '
                     'convergence towards the true distribtion before draws '
                     'are taken.'))
 @click.option('--delay', required=False, default=10,
-              type=click.INT,
+              type=click.INT, show_default=True,
               help=('Number passes between each sampling (draw) of the '
                     'Markov chain. Once the burnin passes have been made, a '
                     'sample will be taken every `delay` number of passes. '
                     'This is also known as `thinning`. Thinning helps reduce '
                     'the impact of correlation between adjacent states of the '
                     'Markov chain.'))
 @click.option('--cluster_start_delay', required=False, default=25,
-              type=click.INT,
+              type=click.INT, show_default=True,
               help=('When using multiple jobs, the script has to start an '
                     '`ipcluster`. If ipcluster does not recognize that it '
                     'has been successfully started, the jobs will not be '
                     'successfully launched. If this is happening, increase '
                     'this parameter.'))
+@click.option('--source_sink_column', required=False, default='SourceSink',
+              type=click.STRING, show_default=True,
+              help=('Sample metadata column indicating which samples should be'
+                    ' treated as sources and which as sinks.'))
+@click.option('--source_column_value', required=False, default='source',
+              type=click.STRING, show_default=True,
+              help=('Value in source_sink_column indicating which samples '
+                    'should be treated as sources.'))
+@click.option('--sink_column_value', required=False, default='sink',
+              type=click.STRING, show_default=True,
+              help=('Value in source_sink_column indicating which samples '
+                    'should be treated as sinks.'))
+@click.option('--source_category_column', required=False, default='Env',
```

Comment:
This flag will also describe the sink samples too. If more than one sample has the same sink id in this column, those two samples will get collapsed. So perhaps renaming to just `category_column` or `env_column`?